packages/extension/package.json

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
Maybe the `packages/extension` folder could be renamed to `packages/blockly-extension` for consistency?

Reply:
Yep, that makes more sense. Thanks